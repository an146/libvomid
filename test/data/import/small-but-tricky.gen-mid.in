#!/bin/sh

source gen-mid.sh

header 2
track <<EOT
	pitchwheel 1 0x2000
	ctrl 10 7 100
	on 1 10 20
	delay 200
	pitchwheel 1 0x2001
	pitchwheel 1 0x1FFF
	ctrl 11 7 101
	off 1 10 20
	pitch 11
	on 1 10 30
	delay 100
	off 2 10 15
	off 2 10 66
	on 2 10 20
	delay 128
	end
EOT
track <<EOT
	notesystem 3
	delay 250
	pitch 16383
	on 2 10 5
	delay 50
	program 9 0
	program 9 1
	program 9 0
	notesystem 4
	off 1 10 35
	pitch 16384
	on 1 10 10
	delay 10
	program 9 123
	pitchwheel 1 0x2001
	meta 01 "xyi"
EOT
