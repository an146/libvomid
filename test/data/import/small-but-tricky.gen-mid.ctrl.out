Tempo:
Time Signature:
Track 0: Name:
Track 0: Notesystem:-1
Track 0: Volume:101
Track 0: Pan:64
Track 0: Balance:
Track 0: Expression:
Track 1: Name:
Track 1: Notesystem:3
Track 1: Volume:100
Track 1: Pan:64
Track 1: Balance:
Track 1: Expression:
Channel 0: Program:
Channel 0: Pitch Wheel:
Channel 1: Program:
Channel 1: Pitch Wheel:200 -1 310 1
Channel 2: Program:
Channel 2: Pitch Wheel:
Channel 3: Program:
Channel 3: Pitch Wheel:
Channel 4: Program:
Channel 4: Pitch Wheel:
Channel 5: Program:
Channel 5: Pitch Wheel:
Channel 6: Program:
Channel 6: Pitch Wheel:
Channel 7: Program:
Channel 7: Pitch Wheel:
Channel 8: Program:
Channel 8: Pitch Wheel:
Channel 9: Program:310 123
Channel 9: Pitch Wheel:
Channel 10: Program:
Channel 10: Pitch Wheel:
Channel 11: Program:
Channel 11: Pitch Wheel:
Channel 12: Program:
Channel 12: Pitch Wheel:
Channel 13: Program:
Channel 13: Pitch Wheel:
Channel 14: Program:
Channel 14: Pitch Wheel:
Channel 15: Program:
Channel 15: Pitch Wheel:
